text/0510-dynamodb-global-table.md

```diff
@@ -0,0 +1,1179 @@
+4. `MULTI_REGION_KEY`: [NEW] A multi region KMS key and its supporting stacks in replica regions will be
+provisioned automatically.
+
+__NOTE__:
+
+* Encryption mode `CUSTOMER_MANAGED` is now deprecated. You can switch to either `KEY_ARNS` or `MULTI_REGION_KEY` mode.
+
+#### Grants
+
+Global table's `replica(region)` method would return an `ITable` reference for the replica in the region. Grant methods
+can be used to provide replica specific permissions.
+
+```typescript
+class FooStack extends Stack {
+  public readonly globalTable: GlobalTable;
+
+  constructor(scope: App, id: string, props: StackProps) {
+    super(scope, id, props);
+
+    this.globalTable = new GlobalTable(this, 'FooTable', {
+      tableName: 'FooGlobalTable',
+      partitionKey: {
+        name: 'FooHashKey',
+        type: AttributeType.STRING,
+      },
+      replicas: [
+        {
+          region: 'us-west-2',
+          encryptionKey: 'FooKmsKeyArn',
+        },
+        {
+          region: 'us-east-1',
+          encryptionKey: 'BarKmsKeyArn',
+        },
+      ],
+    });
+  }
+}
+
+interface BarStackProps extends StackProps {
+  table: ITable;
+}
+
+class BarStack extends Stack {
+  constructor(scope: App, id: string, props: BarStackProps) {
+    super(scope, id, props);
+
+    const user = new iam.User(this, 'User');
+    props.table.grantReadData(user);
+  }
+}
+
+const fooStack = new FooStack(app, 'FooStack', {
+  env: {
+    region: 'us-west-2',
+  },
+});
+
+const barStack = new BarStack(app, 'BarStack', {
+  env: {
+    region: 'us-east-1',
+  },
+  table: fooStack.globalTable.replica('us-east-1'),
+});
+```
+
+#### Metrics
+
+Similar to grant methods, you can access metrics emitted by global table replicas by using `metric` methods.
+
+```typescript
+const globalTable = new GlobalTable(tableStack, 'FooTable', {
+  tableName: 'FooGlobalTable',
+  partitionKey: {
+    name: 'FooHashKey',
+    type: AttributeType.STRING,
+  },
+  replicas: [
+    {
+      region: 'us-west-2',
+    },
+    {
+      region: 'us-east-1',
+    },
+  ],
+});
+
+const replica = globalTable.replica('us-east-1');
+const graphMetric = replica.metricConsumedWriteCapacityUnits();
+
+const dashboard = new Dashboard(
+  tableStack,
+  'Table-Dashboard',
+  {
+    dashboardName: 'Dashboard',
+  },
+);
+
+dashboard.addWidgets(
+  new GraphWidget({
+    title: 'Consumed Write Capacity Units',
+    width: 12,
+    left: [
+      graphMetric,
+    ],
+  }),
+);
+```
+
+#### Importing
+
+You can import an existing global table in your stack by using `from` functions. You will need to specify
+either table name, table arn or table attributes to import it.
+
+```typescript
+GlobalTable.fromTableName(stack, 'FooTableId', 'FooTable');
+```
+
+### NOTE
+
+* We only support version [2019.11.21](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/globaltables.V2.html)
+of the global table since CloudFormation just has support for the same.
```

Comment:
Just a note: "the same" to refer to something earlier in the sentence is a very Indian English thing to say. It's not common in international English.

I'll leave it to you if you want to change it or not, you can be proud of your heritage and let it shine through. But be sure to do it consciously 🤗

Reply:
Ahh, I see. I really appreciate your feedback. I was not aware of it, will update this. :)